apps/citizen-func/utils/redis.ts

```diff
@@ -43,7 +44,7 @@ const createClusterRedisClient = (
         port: redisPort
       }
     ]
-  }) as redis.RedisClient; // Casting RedisClustr with missing typings to RedisClient (same usage).
+  }) as redis.RedisClientType; // Casting RedisClustr with missing typings to RedisClient (same usage).
```

Comment:
we should avoid this kind of casting :'( it is a bad practice, what is preventing you to return the actual redis client type?

Reply:
You are right @lucacavallaro, I also have a consideration here, this function should create a new redis cluster using an old and no longer maintained library [redis-clustr](https://www.npmjs.com/package/redis-clustr).
The correct thing to do at this point (in my opinion) would be to remove this library and use the`redis` library (we have just updated) to create a cluster BUT, i noticed that we are not (and we do not plan to) use a cluster for this function. In fact to create a cluster we should have a couple of redis nodes which we do not have as far as I now.
My proposal is to eliminate the code that deals with cluster management together with the library `redis-clustr` so we can clean up some useless code, what's your take @lucacavallaro @michaeldisaro?

Reply:
Be carefull, I don't think we need redis cluster anymore in this functions... we are using a single redis as a cache, we are not sharding data on a cluster.

Reply:
PS: Just to be clear, I agree with removing redis cluster and using just the new redis library.

Reply:
Removed here [31ae191](https://github.com/pagopa/io-messages/pull/12/commits/31ae1910774b071c4d2beba290dd8ffeddc40d49)

Reply:
LGTM

Reply:
Is the cast still necessary now?

Reply:
I removed the cast together with the `redis-clustr` and the related code.
Resolving this conversation 😄